client_admin/pulp/client/admin/tasks.py

```diff
@@ -31,25 +36,35 @@ class BaseTasksSection(PulpCliSection):
     to provide consistent functionality for a subset of tasks.
     """
 
-    all_flag = PulpCliFlag('--all', _('if specified, all tasks in all states are shown'),
+    all_flag = PulpCliFlag('--all', _('if specified, all tasks in all states '
+                                      'are shown'),
```

Comment:
This change doesn't seem to be necessary. It fit our 100-character line limit before the change.